README.md

```diff
@@ -7,11 +7,11 @@ An RDF library in Scala
 -----------------------
 
 `banana-rdf` is a library for RDF, SPARQL and Linked Data technologies
-in Scala.
+in Scala and ScalaJS.
 
 It can be used with existing libraries without any added cost. There
 is no wrapping involved: you manipulate directly the real objects. We
-currently support Jena, Sesame and Plantain, a pure Scala
+currently support Jena, Sesame, N3js and Plantain, a pure Scala
```

Comment:
N3js should not be here as by "existing libraries" we really mean RDF datastore libraries". So, when `Blazegraph` is added, that would go here.